src/components/ProjectsPage/projects.util.js

```diff
@@ -328,7 +328,7 @@ export const generateMonitoringCounters = (data, dispatch) => {
 
 export const onDeleteProject = (project, setConfirmData, ...args) => {
   setConfirmData({
-    item: project,
+    item: null,
```

Comment:
Names are confusing here.
`item` has 2 meanings: during the first delete it's responsible for defining `deleteNonEmpty` value, but on second delete it's already the actual `project` value. 
Therefore it's not clear what is inside `deleteNonEmpty` parameter.
I think `deleteNonEmpty` parameter can be renamed to `project` in the `confirmHandler`. And don't use it to define `deleteNonEmpty` of the `handleDeleteProject` method. Just pass `false` value.